src/demo/DlTableDemo.vue

```diff
@@ -592,17 +637,17 @@ export default defineComponent({
 }
 
 .sticky-header {
-    ::v-deep .dl-table__top,
-    ::v-deep .dl-table__bottom,
-    ::v-deep thead tr:first-child th {
+    :deep.dl-table__top,
```

Comment:
`:deep` will not work in a vue2 application. please change it back to `::v-deep`